lib/logger.js

```diff
@@ -126,8 +126,8 @@ class Logger extends EventEmitter {
         throw err
       }
       tags = tags
-        .filter((tag) => { return tag !== '' })
-        .map((tag) => { return tag.trim() })
+        .filter(Boolean)
+        .map((tag) => { return String(tag).trim() })
```

Comment:
could be a separate issue, but both versions of the code would allow an empty string. maybe this should map first and reject empty values before filtering, or just do it all in one iteration?

```js
tags = [' ', '\n', '\t']
```

Reply:
ok, good call.  We can map first.